styles/npm/catppuccin.user.css

```diff
@@ -1098,6 +1098,20 @@
       }
     }
 
+    .dark-gray {
+        color: @text !important;
+    }
+
+    .bg-dark-gray, .a11y-light-gray, .bg-gray {
+        background-color: @accent-color !important;
```

Comment:
Why is this accent? Also please theme `.npm-bg-hero` while you are at, you can use a solid color since it is a non-filterable PNG image.

Reply:
ig i'll change it to something else